src/NServiceBus.Core/Settings/ReadOnlySettings.cs

```diff
@@ -83,5 +84,10 @@ public interface ReadOnlySettings
         /// Setup property injection for the given type based on convention.
         /// </summary>
         void ApplyTo<T>(IComponentConfig config);
+        
+        /// <summary>
+        /// Setup property injection for the given type based on convention.
```

Comment:
why r we adding new stuff to support property injection. i thought we were moving away from that?